pyzeebe/errors/pyzeebe_errors.py

```diff
@@ -24,3 +24,16 @@ def __init__(self, task_type: str):
 
 class MaxConsecutiveTaskThreadError(PyZeebeError):
     pass
+
+
+class BusinessError(PyZeebeError):
+    """
+    Exception that can be raised with a user defined code,
+    to be caught later by an error event in the Zeebe process
+    """
+    def __init__(self, error_code: str) -> None:
```

Comment:
Couldn't `BusinessError` also receive the message string? What do you think about that?

Reply:
It could but I'm not sure it's really useful, since Zeebe itself never receive that message (or at least does not take any action on it). But I can add an optional `msg` argument to `BusinessError` constructor anyway that will change the string representation of the `BusinessError` and thus the error message that is sent in the default exception handler (which can than be overridden if the default behaviour needs to be customised).